CliFx.Tests/HelpTextSpecs.cs

```diff
@@ -577,6 +577,51 @@ public class Command : ICommand
             );
         }
 
+        [Fact]
+        public async Task Help_text_shows_all_valid_values_for_list_enum_parameters_and_options()
```

Comment:
```suggestion
        public async Task Help_text_shows_all_valid_values_for_non_scalar_enum_parameters_and_options()
```

The nomenclature used for this is "non-scalar".

Reply:
Thank you, I have made the change and amended my original commit.